darwin/DarwinProcess.c

```diff
@@ -39,9 +39,9 @@ const ProcessFieldData Process_fields[LAST_PROCESSFIELD] = {
    [M_VIRT] = { .name = "M_VIRT", .title = " VIRT ", .description = "Total program size in virtual memory", .flags = 0, .defaultSortDesc = true, },
    [M_RESIDENT] = { .name = "M_RESIDENT", .title = "  RES ", .description = "Resident set size, size of the text and data sections, plus stack usage", .flags = 0, .defaultSortDesc = true, },
    [ST_UID] = { .name = "ST_UID", .title = "UID", .description = "User ID of the process owner", .flags = 0, },
-   [PERCENT_CPU] = { .name = "PERCENT_CPU", .title = "CPU% ", .description = "Percentage of the CPU time the process used in the last sampling", .flags = 0, .defaultSortDesc = true, .autoWidth = true, },
+   [PERCENT_CPU] = { .name = "PERCENT_CPU", .title = " CPU%", .description = "Percentage of the CPU time the process used in the last sampling", .flags = 0, .defaultSortDesc = true, .autoWidth = true, },
    [PERCENT_NORM_CPU] = { .name = "PERCENT_NORM_CPU", .title = "NCPU%", .description = "Normalized percentage of the CPU time the process used in the last sampling (normalized by cpu count)", .flags = 0, .defaultSortDesc = true, .autoWidth = true, },
-   [PERCENT_MEM] = { .name = "PERCENT_MEM", .title = "MEM% ", .description = "Percentage of the memory the process is using, based on resident memory size", .flags = 0, .defaultSortDesc = true, },
+   [PERCENT_MEM] = { .name = "PERCENT_MEM", .title = " MEM% ", .description = "Percentage of the memory the process is using, based on resident memory size", .flags = 0, .defaultSortDesc = true, },
```

Comment:
I think we should discuss if we want to increase the size of the `MEM%` column too. I don't think it's strictly necessary.

Reply:
Actually, this is kind of required. If we don't put a space before and after `MEM%`, either `MEM%` column is misaligned or `▽` character is missing.
  - No left whitespace (`MEM% `)
      Title misaligned with its values.
       ![misaligned-mem](https://user-images.githubusercontent.com/28824347/159642133-f3de432e-1e39-4d74-98ed-afe1214f4762.png)

  - No right whitespace (` MEM%`)
       `▽` character is missing.
       ![missing-characters](https://user-images.githubusercontent.com/28824347/159642720-8ca1daef-83d9-495a-8fe7-1f423d1fcee5.png)

Reply:
The right whitespace is usually used for the direction arrow.

Reply:
@BenBE, how can I align column values with its title without increasing title width of `MEM%` to **6**? I tried different position for white space, but only this combination gives me indented result: `_CPU%` and `_MEM%_`, where `_` represents a white space.

Reply:
The sorting direction is marked in the last column directly after the value. Or to put it a bit different: If you have a column with N characters wide values, there's always another space character added. And the column that holds this space character is also the one that receives the marker for the sorting direction.

In the second screenshot above, actually `Time+` and `Command` start a column to early due to exactly that space character missing after the `MEM%` column heading.